elegant/worm_data.py

```diff
@@ -106,6 +106,16 @@ def get_genotype(path):
             for w in worms:
                 try:
                     w.calculate_ages_and_spans()
+                    for feature_name in dir(w.td):
+                        if feature_name == 'age' or feature_name.endswith('_age'):
+                            feature_vals = getattr(w.td, feature_name)
+                            feature_vals *= age_scale
```

Comment:
If you look at how the `Worm` constructor does this, it explicitly casts the values to a numpy array before trying to multiply -- what if a feature happens to be a list for some reason?